examples/step-74/step-74.cc

```diff
@@ -202,7 +202,7 @@ namespace Step74
                          const double       cell_extend_left,
                          const double       cell_extend_right)
   {
-    const double degree = std::max(1., static_cast<double>(fe_degree));
+    const auto degree = std::max<double>(1, fe_degree);
```

Comment:
I prefer the original statement here. This makes it obvious that a conversion to double happens.